src/irish-cabbage-soup.md

```diff
@@ -0,0 +1,29 @@
+# Irish Cabbage Soup  
+
+Originally from a finnish book _Saiturin Keittiö_ (The Cheapskates Kitchen) from 1992.
+
+- 🍳 Cook time: 45 min
+- 💲  Cost <5€
+
+## Ingredients
+
+- 250g of pork cubes
+- 4-5 midsized potatoes
+- a big chunk of cabbage
+- marjoram
+
+## Directions
+
+1. Cut the cabbage into slices
+2. Peel and cut the potates into cubes
+3. Put some oil into the pot along with the cabbage and pork
+4. Poach in a mild temperature lightly
+5. Add 1,5 liters of water and 10-15 black peppers along with marjoram
```

Comment:
```suggestion
5. Add 1.5 liters of water and 10-15 black peppers along with marjoram
```

Same thing with the comma for decimal. Not sure if we will want to standardize this or not.